README.md

```diff
@@ -0,0 +1,190 @@
+# Charmed Data Integrator Operator
+
+## Overview
+
+This charm allows to retrieve product credentials needed to authenticate (SCRAM) with different kinds of data platform products([MongoDB](https://github.com/canonical/mongodb-operator), [MySQL](https://github.com/canonical/mysql-operator), [PostgreSQL](https://github.com/canonical/postgresql-operator), and [Kafka](https://github.com/canonical/kafka-operator)). 
+
+
+## Config options
+
+The supported configuration options are the following: 
+
+database-name - `string`; The desired database name for which the access will be granted.
+
+topic-name - `string`; The topic name for which the access is granted.
+
+extra-user-roles - `string`; a comma-separated list of values that contains the required extra roles, e.g. `admin` in case of a database or `producer`, `consumer` in case of Kafka. 
```

Comment:
**quibble:** I really like visual representations. Maybe a markdown table here? 

If you like the idea, inspiration could be (just an idea, feel free to ignore):

## Config Options

Different Data charms require different config options. Below are the required config options for each charm:

|            | database-name | topic-name | extra-user-roles |
|------------|---------------|------------|------------------|
| kafka      |               | ✅          | ✅                |
| postgresql | ✅             |            |                  |
| mongodb    | ✅             |            |                  |
| mysql      | ✅             |            |                  |

Reply:
I really like this idea! I added the table after the parameter description